web/packages/teleport/src/Roles/RoleEditor/StandardEditor/AdminRules.tsx

```diff
@@ -194,3 +213,86 @@ function ResourceKindMultiValue(props: MultiValueProps<ResourceKindOption>) {
     </HoverTooltip>
   );
 }
+
+/** Renders a grid of allowed permissions (verbs) as a grid of checkboxes. */
+function VerbEditor({
```

Comment:
Looking at the video https://goteleport.zoom.us/clips/share/xZbZ-dMqTp6rY4Un7sArWg, so with the current way, the verbs applies to all the resources selected in the `Teleport Resources` as long as that resource support those verbs right? 

Will it be confusing since the `node` does not support `rotate`, `readnosecrets`? 
Should we prevent users from grouping resources if they do not support same verbs?

Reply:
@flyinghermit That's a very good question, but my opinion is that if the user selects the resource types one by one (which is the way the UI works), it should be obvious that adding certain resources to the mix extends the available permission set.